src/style.css

```diff
@@ -2,0 +1,51 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
```

Comment:
It's recommended to add a fallback background color that will be visible while the image is loading or if the image fails to load.